libs/recipies/ui-recipies-sidebar/src/lib/search-bar/search-bar.component.html

```diff
@@ -1,4 +1,4 @@
-<form style="display: flex">
+<form class="container">
   <input class="search-bar" type="text" matInput placeholder="Search" />
   <div class="buttons">
```

Comment:
according to BEM we could name this classes this way:
```suggestion
<form class="search-bar">
  <input class="search-bar__input" type="text" matInput placeholder="Search" />
  <div class="search-bar__buttons">
```